.gitignore

```diff
@@ -6,8 +6,8 @@ pyproject.toml
 poetry.lock
 .DS_Store
 examples/screenshots/diffs
-
-# Special for this repo
+examples/offscreen.png
+ Special for this repo
```

Comment:
Think you lost a `#` there

Reply:
Oooh nice catch. I suspect this happened when I edited this file in VI; I'm still struggling how to go into insert mode without an insert key on Mac 😆